frontend/src/pages/orders/OrderList.tsx

```diff
@@ -0,0 +1,118 @@
+import OrderService from "../../services/OrderService.ts";
+import React, {useEffect, useState} from "react";
+import {Order} from "../../types/Order.ts";
+import PageHeader from "../../layouts/PageHeader.tsx";
+import {Link} from "react-router-dom";
+
+const orderService = new OrderService();
+function OrderList() {
+    const [orders, setOrders] = useState<Order[]>([]);
+    const [filter, setFilter] = useState("");
```

Comment:
I would suggest you, to rename this variable to "selectedCustomerName" or something similar. If you use the name "filter" here, one can very easily confuse this variable name with the array-method "filter", that you are also using in line 13: orderByStatus.filter(() => )
If you rename your "filter" variable to a variable name, that actually describes, what the contenttype of the values are, your code is better to read / understand

Reply:
I changed the variable name filter to selectedCustomerName